packages/next/src/utils.ts

```diff
@@ -2731,6 +2760,37 @@ export async function getMiddlewareBundle({
   };
 }
 
+/**
+ * Attempts to read the functions config manifest from the pre-defined
+ * location. If the manifest can't be found it will resolve to
+ * undefined.
+ */
+export async function getFunctionsConfigManifest(
+  entryPath: string,
+  outputDirectory: string
+): Promise<FunctionsConfigManifest | undefined> {
+  const functionConfigManifestPath = path.join(
+    entryPath,
+    outputDirectory,
+    './server/functions-config-manifest.json'
+  );
+
+  const hasManifest = await fs
+    .access(functionConfigManifestPath)
+    .then(() => true)
+    .catch(() => false);
```

Comment:
I think you could use fs-extra's `fs.ensureFile` here: https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/ensureFile.md

Reply:
That would create the file if it doesn't exist. Really this should just be a try/catch around the `fs.readJSON()` call and return on `err.code === 'ENOENT'`.